utils/mysocketio/mysocketio.go

```diff
@@ -0,0 +1,111 @@
+package mysocketio
```

Comment:
lets move this package to upper level, not under utils

Reply:
done